ctparse/rule.py

```diff
@@ -34,7 +34,7 @@
 _regex_year = r'(?:19\d\d)|(?:20[0-2]\d)|(?:\d\d)'
 
 # used in many places in rules
-_regex_to_join = (r'(\-|to( the)?|(un)?til|bis( zum)?|zum|auf( den)?|und|'
+_regex_to_join = (r'(\-|/|to( the)?|(un)?til|bis( zum)?|zum|auf( den)?|und|'
```

Comment:
@sebastianmika  this is mostly to draw your attention to this line. This matches plenty of stuff. This change, along with another that appends a "final" token once the parse is complete.  allows us to gain +2.5 % on intervals but lose -0.005 % on times.